.env.default

```diff
@@ -13,3 +13,6 @@ export TEST_DATABASE_HOST=
 export TEST_DATABASE_NAME=
 export TEST_DATABASE_USER=
 export TEST_DATABASE_USER_PASSWORD=
+
+
+export BRAINSTORMING_TEASERS=true
```

Comment:
We should consider these entries as well.

I tried out the following edge cases: 
- `export BRAINSTORMING_TEASERS=`<- skipped the feature because of empty string
- `export BRAINSTORMING_TEASERS="     true        "` <- skipped the feature because of included whitespaces.